seekinternational/seek-asia-style-guide
# fix(Checkbox): Putting label at root level so everything is clickable

Checkbox isn't clickable on the actual checkbox. This isn't great.


### To Test:
Go here: https://seekinternational.github.io/seek-asia-style-guide/checkbox

Click on the label, and see it select and unselect. Now click on the checkbox, does it work? It doesn't for me.

Now, load up this PR, and do the same thing. Hopefully clicking on the actual checkbox should work again.
